src/main/java/com/bigbookmarket/domain/Item.java

```diff
@@ -0,0 +1,41 @@
+package com.bigbookmarket.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Item {
+
+    @Id
+    @Column(columnDefinition = "BIGINT")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long itemId;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "bookId", referencedColumnName = "bookId")
+    private Book book;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "sellerId", referencedColumnName = "userId")
+    private User user;
+
+    @Column(columnDefinition = "MEDIUMINT", nullable = false)
+    private Long price;
```

Comment:
```suggestion
    private Integer price;
```

저희는 가격이니까 unsigned이긴 한데 signed Integer 최댓값 넘을 일 없을 것 같아서 그냥 Integer로 하는게 좋을 것 같아요~